README.md

```diff
@@ -153,6 +159,8 @@ $ python driver.py --device stm32f4dis --os nuttx --app iotjs --port /dev/ttyACM
 $ python driver.py --device stm32f4dis --os nuttx --app jerryscript --port /dev/ttyACM0 --baud 115200
 $ python driver.py --device rpi2 --os linux --app iotjs --address a.b.c.d --username pi --remote-path /home/pi/testrunner
 $ python driver.py --device rpi2 --os linux --app jerryscript --address a.b.c.d --username pi --remote-path /home/pi/testrunner
+$ python driver.py --device artik053 --os tizenrt --app iotjs --port /dev/ttyUSB1 --baud 115200
+$ python driver.py --device artik053 --os tizenrt --app jerryscript --port /dev/ttyUSB1 --baud 115200
```

Comment:
I think this line can be misleading if we don't support this os-app combination yet.

Reply:
I removed this line.